src/configs/recommended.ts

```diff
@@ -0,0 +1,127 @@
+/**
+ * @license
+ * Copyright 2016 Palantir Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const rules = {
+    "align": [true,
+        "parameters",
+        "statements",
+    ],
+    "class-name": true,
+    "comment-format": [true,
+        "check-space",
+    ],
+    "curly": true,
+    "eofline": true,
+    "forin": true,
+    "indent": [true, "spaces"],
+    "interface-name": true,
+    "jsdoc-format": true,
+    "label-position": true,
+    "label-undefined": true,
+    "max-line-length": [true, 120],
+    "member-access": true,
+    "member-ordering": [true, {
+        "order": "statics-first",
+    }],
+    "new-parens": true,
+    "no-any": false,
+    "no-arg": true,
+    "no-bitwise": true,
+    "no-conditional-assignment": true,
+    "no-consecutive-blank-lines": true,
+    "no-console": [true,
+        "debug",
+        "info",
+        "log",
+        "time",
+        "timeEnd",
+        "trace",
+    ],
+    "no-construct": true,
+    "no-constructor-vars": true,
```

Comment:
I think we should disable this one, it's a legit TS feature for removing redundancy in code.

Also, they're officially called "parameter property declarations", so this rule has a sort of unclear name as well